apps/document-publish-function/publish-document/publish-document.js

```diff
@@ -1,16 +1,30 @@
 import { BlobStorageClient } from '@pins/blob-storage-client';
+import { getEventClient } from '@pins/event-client';
 import config from './config.js';
 
+/**
+ *
+ * @param {object} event
+ * @param {import('@azure/functions').Context} context
+ */
+const sendEvent = async (event, context) => {
+	const eventClient = getEventClient(
```

Comment:
Can we create this client outside of the sendEvent method? Would be nice to re-use it so we don't have to keep getting fresh credentials